src/databases/OpenPMD/OpenPMDClasses/PMDFile.C

```diff
@@ -165,6 +175,16 @@ void PMDFile::ScanFileAttributes()
     // OpenPMD files always contain a data group at the root
     groupId = H5Gopen(fileId, "/",H5P_DEFAULT);
```

Comment:
As I understand HDF5, all HDF5 files have a root grouped named, `"/"` and so this should *never* fail. Put another way, this is a useless test. 

I am thinking that we should be testing for some specific group name here like maybe `"data"` or something? I don't know because I don't know anything about the OpenPMD format. 

I am waiting for a `git lfs pull` to retreive the `openpmd_test_data.7z` file so I can take a quick look and suggest something better